packages/apps/graph-client/src/components/compact-transfers-table/compact-transfers-table.tsx

```diff
@@ -56,72 +81,76 @@ export const CompactTransfersTable = (
+              : transfer.height;
 
             return (
               <Table.Tr key={index}>
                 <Table.Td>{chainIdDisplay}</Table.Td>
                 <Table.Td>{heightDisplay}</Table.Td>
-                <Table.Td>{edge?.node.amount}</Table.Td>
+                <Table.Td>{transfer.amount}</Table.Td>
                 <Table.Td>
                   <Link
-                    href={`${routes.ACCOUNT}/${moduleName}/${edge?.node.senderAccount}`}
+                    href={`${routes.ACCOUNT}/${moduleName}/${transfer.senderAccount}`}
                   >
-                    <span title={edge?.node.senderAccount}>
-                      {truncate(edge?.node.senderAccount)}
+                    <span title={transfer.senderAccount}>
+                      {truncate(transfer.senderAccount)}
                     </span>
                   </Link>
                 </Table.Td>
                 <Table.Td>
-                  {!edge?.node.crossChainTransfer ? (
+                  {!crossChainCounterPart ? (
                     <Link
-                      href={`${routes.ACCOUNT}/${moduleName}/${edge?.node.receiverAccount}`}
+                      href={`${routes.ACCOUNT}/${moduleName}/${transfer.receiverAccount}`}
                     >
-                      <span title={edge?.node.receiverAccount}>
-                        {truncate(edge?.node.receiverAccount)}
+                      <span title={transfer.receiverAccount}>
+                        {truncate(transfer.receiverAccount)}
                       </span>
                     </Link>
                   ) : (
                     <Link
-                      href={`${routes.ACCOUNT}/${moduleName}/${edge?.node.crossChainTransfer.receiverAccount}`}
+                      href={`${routes.ACCOUNT}/${moduleName}/${crossChainCounterPart.receiverAccount}`}
                     >
-                      <span
-                        title={edge?.node.crossChainTransfer.receiverAccount}
-                      >
-                        {truncate(
-                          edge?.node.crossChainTransfer.receiverAccount,
-                        )}
+                      <span title={crossChainCounterPart.receiverAccount}>
+                        {truncate(crossChainCounterPart.receiverAccount)}
                       </span>
                     </Link>
                   )}
                 </Table.Td>
                 <Table.Td>
-                  <Link
-                    href={`${routes.TRANSACTIONS}/${edge?.node.requestKey}`}
-                  >
-                    <span title={edge?.node.requestKey}>
-                      {truncate(edge?.node.requestKey)}
+                  <Link href={`${routes.TRANSACTIONS}/${transfer.requestKey}`}>
+                    <span title={transfer.requestKey}>
+                      {truncate(transfer.requestKey)}
                     </span>
                   </Link>
                   /
```

Comment:
only add the / if there is a crossChainCounterPart

Reply:
Fixed